CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.3.2] - 2016-05-12
```

Comment:
Please use the CHANGELOG in master as your starting point and add the 1.3.2 release info to that.

https://github.com/cisco/cisco-network-puppet-module/blob/master/CHANGELOG.md